### PR TITLE
ci: add a patch that don't check FD for STDIN/STDOUT/STDERR

### PR DESCRIPTION
**Pull request by komainu8**
This is testing now.

### CODE REVIEW

**Comment by kou** [comment]
The patch should be applied before `configure` or `cmake`.

**Comment by komainu8** [comment]
I see.
I'll fix it.

File: .github/workflows/linux-mingw.yml [hunk 1]
```diff
@@ -44,6 +44,8 @@ jobs:
       - name: Generate configure
         run: |
           ./autogen.sh
+      - name: Apply a patch
+          patch -p1 < packages/source/patches/mruby-do-not-check-fd-for-stdin-stdout-stderr.diff
```
Comment (kou) [inline .github/workflows/linux-mingw.yml:48]: `run:` is missing. This breaks CI.

File: .github/workflows/linux.yml [hunk 2]
```diff
@@ -88,6 +88,8 @@ jobs:
       - name: Generate configure
         run: |
           ./autogen.sh
+      - name: Apply a patch
```
Comment (kou) [inline .github/workflows/linux.yml:91]: `run:` is missing. This breaks CI.

File: .github/workflows/macos.yml [hunk 3]
```diff
@@ -31,6 +31,8 @@ jobs:
           matrix.use-autotools
         run: |
           ./autogen.sh
+      - name: Apply a patch
```
Comment (kou) [inline .github/workflows/macos.yml:34]: `run:` is missing. This breaks CI.

File: .github/workflows/windows-mingw.yml [hunk 4]
```diff
@@ -28,10 +28,13 @@ jobs:
             mingw-w64-x86_64-ruby
             mingw-w64-x86_64-xxhash
             mingw-w64-x86_64-zlib
+            patch
       - name: Generate configure
         shell: msys2 {0}
         run: |
           ./autogen.sh
+      - name: Apply a patch
```
Comment (kou) [inline .github/workflows/windows-mingw.yml:36]: `run:` is missing. This breaks CI.

**Comment by komainu8** [comment]
Thank you for your comment.
I am aware of it. I am fixing them now.

**Pull request merged**